redash/query_runner/big_query.py

```diff
@@ -156,6 +161,9 @@ def _get_project_id(self):
     def _get_location(self):
         return self.configuration.get("location")
 
+    def _get_preview_max_results(self):
+        return self.configuration.get("previewMaxResults", 100)
```

Comment:
looks like you should be using the default value you added in line 132.

Reply:
This is mainly here to ensure datasources that were registered before the config schema change won't break until you re-save the settings to set the default, but ok, I can remove that.